contributors/design-proposals/storage/cloning.md

```diff
@@ -44,6 +44,105 @@ Specific use cases around cloning are included in the use cases listed below as
 ## Design Overview
 The proposed design is based on the idea of leveraging well defined concepts for storage in Kubernetes. The cloning will be initiated 
 by creating an annotated PVC to create a new volume using a defined storage class that supports cloning.
-The actual cloning process must be implemented by the provisioning storage and properly understand and react to the annotation to clone.
+The actual cloning process must be implemented by the provisioning storage and properly understand and react to the annotation to clone. In the absence of a storage class that supports cloning a host-assisted clone can also be exected to create the copy. The host-assisted cloning deploys a controller to utlize pod streaming to execute the copy.
```

Comment:
s/and properly understand and react/, therefore plugins that support cloning need to ensure they're checking ..../  

Also, do we want annotations mentioned here or aren't we using VolumeContentSource from the spec?

minor typo s/exected/executed/